module/VuFind/src/VuFind/ILS/Driver/SierraRest.php

```diff
@@ -1179,6 +1215,21 @@ public function getMyHolds($patron)
                 'updateDetails' => $updateDetails,
             ];
         }
+        $n = 0;
+        foreach ($holds as $hold) {
+            if ($this->config['InnReach']['enabled']) {
```

Comment:
I would move the config check outside the foreach loop so you don't iterate at all when the feature is disabled. Also, same need for fallback to default false.